### PR TITLE
SIMPLY-3578: Fixed crashes on iOS 14.5 beta when generating cover from book with non ASCII characters

### PR DESCRIPTION
**Pull request by singhraman4282**
**What's this do?**
Fixes crashes on iOS 14.5 beta when generating cover from book with non ASCII characters

**Why are we doing this? (w/ JIRA link if applicable)**
https://jira.nypl.org/browse/SIMPLY-3578

**How should this be tested? / Do these changes have associated tests?**
The original crash was found on iOS devices running iOS 14.5 beta while displaying books with no covers that have accented letters in their title (https://cdncms.fonts.net/images/de7722aa7a6beee0/D.Accents.jpg). To test the fix, try viewing such books in the list on a device running iOS 14.5 beta. The app should not crash while viewing those books.

**Dependencies for merging? Releasing to production?**
n/a

**Does this include changes that require a new SimplyE build for QA?**
Yes

**Has the application documentation been updated for these changes?**
n/a

**Did someone actually run this code to verify it works?**
Raman Singh tested on iOS 14.5 beta and iOS 13.7

### CODE REVIEW

File: SimplifiedTests/SpecialChars.json [hunk 1]
```diff
@@ -0,0 +1 @@
+†¤¶§†!"#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\]^_`abcdefghijklmnopqrstuvwxyz{|}~ÇüéâäàåçêëèïîìæÆôöòûùÿ¢£¥PƒáíóúñÑ,,¿¬½¼¡«»¦ßµ±°•·²€„…†‡ˆ‰Š‹Œ‘’“”–—˜™š›œŸ¨©®¯³´¸¹¾ÀÁÂÃÄÅÈÉÊËÌÍÎÏÐÒÓÔÕÖ×ØÙÚÛÜÝÞãðõ÷øüýþ
```
Comment (ettore) [inline SimplifiedTests/SpecialChars.json:1]: this file is not a json, can you just change the extension to (e.g.) `txt`?

**Review by ettore — COMMENTED** [review]
thorough unit tests, thanks -- 

I think we are missing a PR for `tenprintcover` though right? also, currently SimplyE points to release `1.0.1` for `tenprintcover`, which will need to be updated after we merge that PR there.

File: SimplifiedTests/SpecialChars.json (hunk 1 above)
Comment (singhraman4282) [reply SimplifiedTests/SpecialChars.json]: Great catch. Done.

**Review by ErnestFan — APPROVED** [review]

File: Simplified.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -6551,6 +6566,7 @@
 				CODE_SIGN_ENTITLEMENTS = Simplified/SimplyE.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CURRENT_PROJECT_VERSION = 7;
+				CURRENT_PROJECT_VERSION = 6;
```
Comment (ettore) [inline Simplified.xcodeproj/project.pbxproj:6569]: I think Xcode messed up here :) can you remove one of these?

**Review by ettore — APPROVED** [review]
GitHub is showing the updates from TenPrintCover in a weird way (usually it just shows the updated commit on the sub repo) but things do seem normal on your end, so let's monitor the build log and see if the sub-repo changes are being picked up correctly.

**Review by vladimirfedorov — APPROVED** [review]

**Comment by ettore** [comment]
> let's monitor the build log and see if the sub-repo changes are being picked up correctly.

I do see your new file that you added being compiled here, so we should be good. Please also take a look though: https://github.com/NYPL-Simplified/Simplified-iOS/runs/2124108615?check_suite_focus=true

File: Simplified.xcodeproj/project.pbxproj (hunk 2 above)
Comment (singhraman4282) [reply Simplified.xcodeproj/project.pbxproj]: Fixed

**Pull request merged**